test/infrastructure/docker/examples/machine-pool.yaml

```diff
@@ -22,14 +22,20 @@ spec:
     name: my-cluster
     namespace: default
 ---
```

Comment:
fyi. I synced the examples so they are all very similar now (including ordering). I think that's the only way we can really maintain them over time